### PR TITLE
New sample using an M5Stack display as a remote controller

### PR DESCRIPTION
**Pull request by pgrawehr**
This provides a larger sample for the features added in #2117, #2104, #2092 and #2093.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/2152)

### CODE REVIEW

File: samples/M5StackRemoteDisplay/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Mirror the screen to an external display
+
+This larger sample mirrors the screen (or parts of it) from Windows or Linux onto an external, small display. The sample is configured to talk to an M5Though IoT module, a readily available integrated piece of hardware from M5Stack. The device features a QVGA display with touchscreen, a power distribution unit connected to an ESP32 microcontroller.
+
+To run the sample, load the [ConfigurableFirmata](https://github.com/firmata/ConfigurableFirmata) Firmware (V3.0 or later) onto the ESP32 first. The tool has some command line options:
+
```
Comment (Ellerbach) [inline samples/M5StackRemoteDisplay/README.md:6]: maybe add a picture of the result?

File: samples/M5StackRemoteDisplay/RemoteControl.cs [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+
+        private void OnNewSequence(NmeaSinkAndSource nmeaSinkAndSource, NmeaSentence nmeaSentence)
+        {
+            // Nothing to do here, handled by the message cache (doing this floods the output)
+            // Console.WriteLine($"Received sentence: {nmeaSentence.ToString()}");
+        }
+
+        private void OnTouched(object o, Point point)
+        {
+            Console.WriteLine($"Touched screen at {point}");
```
Comment (Ellerbach) [inline samples/M5StackRemoteDisplay/RemoteControl.cs:182]: are the Console.WriteLine really needed or could be replaced by Debug.WriteLine? Or just the logger?

**Review by Ellerbach — APPROVED** [review]
Very minor comments, very nice and complete sample!

File: samples/M5StackRemoteDisplay/RemoteControl.cs (hunk 2 above)
Comment (pgrawehr) [reply samples/M5StackRemoteDisplay/RemoteControl.cs:182]: It's just debug output, yes, but the entire console is showing nothing but debug output anyway.

Comment (Ellerbach) [reply samples/M5StackRemoteDisplay/RemoteControl.cs:182]: yes and no :-) with Debug.WriteLine, you do not see those when you build in Release! So it's from my point of view, much cleaner than using the Console.

**Pull request merged**